### ISSUE TITLE
[QUEST][Classic] Gerenzo's Orders

### ISSUE THREAD

**Issue by sinjer** [dalaranwow/dalaran-wow#4536]
Took quest.  Wowhead states it takes about 5m10s for him to mine the mysterious ore.  Meanwhile mobs should be attempting to attack him.  No mobs appeared other than the normal ones in the cave, and 10 minutes later he hasn't finished mining.  This takes place, starts and ends, in Windshear Mine, Stonetalon.

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/590 (<- Klick on the linked number)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
